module/x/gravity/keeper/keeper_valset_test.go

```diff
@@ -15,6 +16,11 @@ const (
 	LastSlashedNonce = 10
 )
 
+func TestAbsolutePower(t *testing.T) {
+	pwr := normalizeValidatorPower(1, math.NewInt(1))
+	require.Equal(t, pwr, uint64(4294967296))
```

Comment:
Where  `4294967296` comes from? Could you add a comment?

Reply:
Will do, it's 2^32 which is how Gravity defines the 100% of the power.

Reply:
can we have one validator (zero addr) with zero power?

Reply:
No, as that would throw an error on the Gravity contract:

```solidity
	// The number of 'votes' required to execute a valset
	// update or batch execution, set to 2/3 of 2^32
	uint256 constant constant_powerThreshold = 2863311530;
	
		// Check cumulative power to ensure the contract has sufficient power to actually
		// pass a vote
		uint256 cumulativePower = 0;
		for (uint256 i = 0; i < _newValset.powers.length; i++) {
			cumulativePower = cumulativePower + _newValset.powers[i];
			if (cumulativePower > constant_powerThreshold) {
				break;
			}
		}
		if (cumulativePower <= constant_powerThreshold) {
			revert InsufficientPower({
				cumulativePower: cumulativePower,
				powerThreshold: constant_powerThreshold
			});
		}
```

So giving 100% to the zero address is the way to go. We could give it 67% or whatever, it doesn't matter really

Reply:
let's link this code in the comment as well. Thanks!